deploy/config/dev_umccr.toml

```diff
@@ -1,3 +1,5 @@
+auth_required = true
+
```

Comment:
I like this idea, I think it simplifies the CDK config. It might be good to document/note that now there are config options that aren't really used directly by htsget-rs, but are rather just CDK-related deployment options. Also need to check that unknown options are definitely allowed when passed to htsget-rs (as htsget-rs won't recognise these).